src/sdk/index.ts

```diff
@@ -419,36 +343,97 @@ export async function createPlayground(
  * @return {string} - The URL of the playground (as a string).
  */
 export function getPlaygroundUrl(options: EmbedOptions = {}): string {
-  const { appUrl, params, config, import: x, ...otherOptions } = options;
-  const configParam =
-    typeof config === 'string'
-      ? { config }
-      : config && typeof config === 'object' && Object.keys(config).length
-        ? { x: 'code/' + compressToEncodedURIComponent(JSON.stringify(config)) }
-        : {};
-
-  let encodedParams;
+  const {
+    appUrl = 'https://livecodes.io',
+    params = {},
+    config = {},
+    headless,
+    import: importFrom,
+    lite,
+    loading = 'lazy',
+    template,
+    view,
+  } = options;
```

Comment:
It would be convenient to add `...otherOptions` here, which would be added to search params, so that we can add more options later without having to change this.